### PR TITLE
Create calibration.rst

### PR DESCRIPTION
**Pull request by LykleSchepers**

### CODE REVIEW

File: docs/nimble_dual/calibration.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* Rotate the paper and see around which nozzle it rotates.
+	- If it is the first nozzle, lower the second nozzle a little more
+	- If it is the second nozzle, raise it a bit.
+
+The idea is to get the paper to rotate around each nozzle with the same force needed. As Richard said: "This is fiddly and time consuming but can be done! "
+
+You have now set the extrusion rate of the Dual Nimble correctly and have the nozzles set right.
+You can now start Using the Dual Nimble.
+
+See the :doc:`Using the Dual Nimble<./using>` page or click Next.
```
Comment (BrianGilbert) [inline docs/nimble_dual/calibration.rst]: Loks like asyntax error here </using>

**Review by BrianGilbert — CHANGES_REQUESTED** [review]
Looks like a syntax issue here

**Review by LykleSchepers — COMMENTED** [review]
I seem to have a problem fixing the link to the next page. Can you look?

**Comment by LykleSchepers** [comment]
Changed the link syntax and it should work now

**Pull request merged**